rs/registry/admin/src/main.rs

```diff
@@ -1215,6 +1218,41 @@ impl ProposalPayload<SubnetRentalRequest> for ProposeToRentSubnetCmd {
         }
     }
 }
+/// Sub-command to submit a subnet rental request proposal.
+#[derive_common_proposal_fields]
+#[derive(Parser, ProposalMetadata)]
+struct ProposeToFulfillSubnetRentalRequestCmd {
+    /// The principal ID of the user whose rental request should be fulfilled
+    #[clap(long)]
+    user: Option<PrincipalId>,
+
+    /// Node IDs that will be members of the subnet
+    #[clap(long, num_args(1..))]
+    node_ids: Option<Vec<PrincipalId>>,
+
+    /// Replica version ID (40 character hexadecimal git commit ID)
+    #[clap(long)]
+    replica_version_id: Option<String>,
```

Comment:
To make ic-admin a little bit easier to use, in the future, I think it would be nice if users could do `--replica-version-id=LATEST`, and then, ic-admin would pick the latest blessed replica_version_id.

Can you add a TODO to this effect?

Reply:
I don't think this should be a todo, as there are other places where replica is specified, so the teams using these tools already have a means to do this.  It's a good idea, but I don't think it will actually be done any time soon (unless you're planning to, in which case open a PR?)